### PR TITLE
clang-tidy: renames to WebKit style

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

**Comment by neheb** [comment]
This thing uncovered an annoying issue. Because of the way macros work with compilers, clang-tidy is unable to follow all paths. I think these macros should be replaced with if constexpr statements.

File: src/util/tools.cc [hunk 1]
```diff
@@ -242,9 +242,9 @@ fs::path findInPath(const fs::path& exec)
     if (!p)
         return {};
 
-    std::string PATH = p;
+    std::string path = p;
     std::error_code ec;
```
Comment (KarlStraussberger) [inline src/util/tools.cc:246]: You need to select another name here, e.g. `pathString`. path is the loop variable already.

File: src/metadata/matroska_handler.cc [hunk 2]
```diff
@@ -52,17 +52,17 @@
 // file managment
 class FileIOCallback : public IOCallback {
 private:
-    std::FILE* file;
+    std::FILE* m_file;
```
Comment (KarlStraussberger) [inline src/metadata/matroska_handler.cc:55]: We don't use that style for attributes. Maybe call it `mediaFile`

**Comment by neheb** [comment]
Fixed.

**Comment by neheb** [comment]
Rebased again.

**Comment by KarlStraussberger** [comment]
Thank you. Really great job.

**Pull request merged**